openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -2623,6 +2652,8 @@ struct AMDGPUDeviceTy : public GenericDeviceTy, AMDGenericDeviceTy {
         });
   }
 
+  bool getUseMultipleSdmaEngines() { return OMPX_UseMultipleSdmaEngines; }
```

Comment:
Having this be a `getter` is weird, maybe just `useMultipleSdmaEngines`.

Reply:
That's a good point. I also added `const` here.